fidesctl/src/fidesapi/main.py

```diff
@@ -67,10 +68,29 @@ async def log_request(request: Request, call_next: Callable) -> Response:
     return response
 
 
-@app.get("/health", tags=["Health"])
+@app.get(
+    "/health",
+    response_model=Dict[str, str],
+    responses={
+        status.HTTP_200_OK: {
+            "content": {
+                "application/json": {
+                    "example": {
+                        "status": "healthy",
+                        "version": "1.0.0",
+                    }
+                }
+            }
+        }
+    },
+    tags=["Health"],
+)
 async def health() -> Dict:
     "Confirm that the API is running and healthy."
-    return {"data": {"message": "Fidesctl API service is healthy!"}}
+    return {
+        "status": "healthy",
+        "version": str(fidesctl.__version__),
+    }
```

Comment:
I didn't see any reason for the top-level `"data"` key, so I removed it in order to flatten the response and ease the usage for any consumers. If this needs to be reverted for some reason, we can just remove my most recent commit, or `git revert` it.

Reply:
thank you!